src/sidebars/search/search.html

```diff
@@ -18,7 +21,7 @@
                   </div>
               </div>
           </div>
-          <div class="no-results" ng-show="searchCtrl.noResults">
+          <div class="results-message no-results" ng-show="searchCtrl.noResults">
```

Comment:
No, but I wanted to keep it so there's at least some differentiation between the two messages.